recipes/crow/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,22 @@
+#include "jwt/jwt.hpp"
```

Comment:
IS it correct? I see "crow.h" as header for include: 

https://github.com/ipkn/crow/blob/master/examples/helloworld.cpp

Reply:
good catch, I've just used another project as template and forgot to update the example. Just updated the test